rest/route/project.go

```diff
@@ -570,6 +571,9 @@ func (h *projectIDPatchHandler) Run(ctx context.Context) gimlet.Responder {
 
 		queue := evergreen.GetEnvironment().RemoteQueue()
 		if err = queue.Put(ctx, j); err != nil {
+			if db.IsDuplicateKey(err) {
```

Comment:
Is it necessary to return this error, or should it just continue on duplicate key error? If the remote queue already has a pending job to run the repotracker on this revision, it seems like the route could just treat it a success since the work will be done eventually.

Reply:
Thanks! updated.